django_fsm_log/south_migrations/0002_auto__add_field_statelog_description.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'StateLog.description'
+        db.add_column(u'django_fsm_log_statelog', 'description',
+                      self.gf('django.db.models.fields.CharField')(max_length=255, null=True, blank=True),
+                      keep_default=False)
+
+
+    def backwards(self, orm):
+        # Deleting field 'StateLog.description'
+        db.delete_column(u'django_fsm_log_statelog', 'description')
+
+
+    models = {
+        u'contenttypes.contenttype': {
+            'Meta': {'ordering': "('name',)", 'unique_together': "(('app_label', 'model'),)", 'object_name': 'ContentType', 'db_table': "'django_content_type'"},
+            'app_label': ('django.db.models.fields.CharField', [], {'max_length': '100'}),
+            u'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
+            'model': ('django.db.models.fields.CharField', [], {'max_length': '100'}),
+            'name': ('django.db.models.fields.CharField', [], {'max_length': '100'})
+        },
+        u'django_fsm_log.statelog': {
+            'Meta': {'object_name': 'StateLog'},
+            'by': ('django.db.models.fields.related.ForeignKey', [], {'to': u"orm['auth.Usuario']", 'null': 'True', 'blank': 'True'}),
```

Comment:
use the tricks from the initial migration to get the `by` field to hook up correctly https://github.com/gizmag/django-fsm-log/blob/master/django_fsm_log/south_migrations/0001_initial.py#L46